reference-architecture/azure-ansible/node.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+#yum -y update
+yum -y install wget git net-tools bind-utils iptables-services bridge-utils bash-completion docker
+
+sed -i -e "s#^OPTIONS='--selinux-enabled'#OPTIONS='--selinux-enabled --insecure-registry 172.30.0.0/16'#" /etc/sysconfig/docker
+                                                                                         
+cat <<EOF > /etc/sysconfig/docker-storage-setup
+DEVS=/dev/sdc
+VG=docker-vg
+EXTRA_DOCKER_STORAGE_OPTIONS="--storage-opt dm.basesize=3G"
+EOF
+
+docker-storage-setup                                                                                                                                    
```

Comment:
Id let the installer handle storage setup and the enabling of docker. This may be a good location though to do you mounts